### PR TITLE
Gh iss 32 dspace connection test command

### PR DESCRIPTION
**Pull request by ehanson8**
#### Includes new or updated dependencies?
YES

#### Changes expectations for external applications?
NO

#### Developer
- [x] All new ENV is documented in README
- [x] All new ENV has been added to staging and production environments
- [x] All related Jira tickets are linked in commit message(s)

#### How can a reviewer manually see the effects of these changes?
Using `dev1` credentials, run `make verify-dspace-connection-dev` and view the logs of the ECS task to see that the connection was successful.

#### Code Reviewer

- [ ] The commit message is clear and follows our guidelines
      (not just this pull request message)
- [ ] There are appropriate tests covering any new functionality
- [ ] The documentation has been updated or is unnecessary
- [ ] The changes have been verified
- [ ] New dependencies are appropriate or there were no changes

### CODE REVIEW

**Comment by ehanson8** [comment]
Closes #33

File: Makefile [hunk 1]
```diff
@@ -46,12 +46,21 @@ publish-stage: ## Only use in an emergency
 run-dev:  ## Runs the task in dev - see readme for more info
 	aws ecs run-task --cluster DSS-SubmissionService-dev --task-definition DSS-SubmissionService-dev-task --network-configuration "awsvpcConfiguration={subnets=[subnet-0488e4996ddc8365b,subnet-022e9ea19f5f93e65],securityGroups=[sg-044033bf5f102c544],assignPublicIp=DISABLED}" --launch-type FARGATE --region us-east-1
 
+verify-dspace-connnection-dev: # Verify dev app can connect to DSpace
```
Comment (ghukill) [inline Makefile:49]: I think small typo here, triple "n", maybe should be: `verify-dspace-connection-dev`

File: Makefile [hunk 2]
```diff
@@ -46,12 +46,21 @@ publish-stage: ## Only use in an emergency
...
+	aws ecs run-task --cluster DSS-SubmissionService-dev --task-definition DSS-SubmissionService-dev-task --network-configuration "awsvpcConfiguration={subnets=[subnet-0488e4996ddc8365b,subnet-022e9ea19f5f93e65],securityGroups=[sg-044033bf5f102c544],assignPublicIp=DISABLED}" --launch-type FARGATE --region us-east-1 --overrides '{"containerOverrides": [ {"name": "DSS", "command": ["verify-dspace-connection"]}]}'
+
 run-stage:  ## Runs the task in stage - see readme for more info
 	aws ecs run-task --cluster DSS-SubmissionService-stage --task-definition DSS-SubmissionService-stage-task --network-configuration "awsvpcConfiguration={subnets=[subnet-05df31ac28dd1a4b0,subnet-04cfa272d4f41dc8a],securityGroups=[sg-0f64d9a1101d544d1],assignPublicIp=DISABLED}" --launch-type FARGATE --region us-east-1
 
+verify-dspace-connnection-stage: # Verify stage app can connect to DSpace
```
Comment (ghukill) [inline Makefile:55]: Same here, probably should be: `verify-dspace-connection-stage`

File: submitter/cli.py [hunk 3]
```diff
@@ -87,3 +89,22 @@ def create_queue(name):
     """Create queue with NAME supplied as argument"""
     queue = create(name)
     logger.info(queue.url)
+
+
+@main.command()
+def verify_dspace_connection():
+    client = DSpaceClient(CONFIG.DSPACE_API_URL, timeout=CONFIG.DSPACE_TIMEOUT)
```
Comment (ghukill) [inline submitter/cli.py:96]: I don't think this needs addressing now, as it's currently a topic of active discussion, but I think the `config.Class` might benefit from raising exceptions if required env vars are not present.

When I attempted to run the following, to test this new command:
```shell
WORKSPACE=dev pipenv run submitter verify-dspace-connection
```

I receieved this
```python
...
File "<PATH>/dspace-submission-service/submitter/cli.py", line 96, in verify_dspace_connection
  client = DSpaceClient(CONFIG.DSPACE_API_URL, timeout=CONFIG.DSPACE_TIMEOUT)
File "<VENV>/lib/python3.9/site-packages/dspace/client.py", line 39, in __init__
  self.base_url: str = base_url.rstrip("/")
AttributeError: 'NoneType' object has no attribute 'rstrip'
```

I can see quite clearly, after the fact, that the README indicates these need to be set.  But I think this gets at a subtle question we may want to answer: what if _all_ env vars don't need to be set for certain actions?  Is it possible for these error messages to indicate what's missing?

Again, not requesting changes here, just noting for our ongoing discussions.

**Review by ghukill — CHANGES_REQUESTED** [review]
Requesting changes on a couple of small typos, and left a comment (but not changes required) about `config.Config`.  Otherwise, looks good!

File: submitter/cli.py (hunk 3 above)
Comment (ehanson8) [reply submitter/cli.py:96]: Great point that I'll consider with work I'm doing on `wiley-deposits`

**Comment by ehanson8** [comment]
Good catch, fixed!

**Review by ghukill — APPROVED** [review]
Looks good!

**Pull request merged**